deploy-workflow/workflows/modules/.github/workflows/main.yml

```diff
@@ -60,15 +60,38 @@ jobs:
       - name: SonarCloud End
         uses: VirtoCommerce/vc-github-actions/sonar-scanner-end@master
 
+      - name: Quality Gate
+        uses: VirtoCommerce/vc-github-actions/sonar-quality-gate@master
+        with:
+          login: ${{secrets.SONAR_TOKEN}}
+
       - name: Packaging
         run: vc-build Compress -skip Clean+Restore+Compile+Test
       
-      # Step runs only on commit to master or dev brunch
       - name: Publish Nuget
         uses: VirtoCommerce/vc-github-actions/publish-nuget@master
```

Comment:
Add check `if: ${{ github.ref == 'refs/heads/dev' || github.ref == 'refs/heads/master'}}`
In VirtoCommerce/vc-github-actions/publish-nuget remove check `if(branchName === 'dev' || branchName === 'master')`